src/scripts/main.js

```diff
@@ -22,7 +22,24 @@ const food = {
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  function rec(obj) {
+    const ul = document.createElement('ul');
+
+    for (const key in obj) {
+      const listItem = document.createElement('li');
+
+      listItem.innerHTML = key;
+
+      const childrenUl = rec(obj[key]);
+
+      listItem.append(childrenUl);
+      ul.append(listItem);
+    }
+
+    return ul;
+  }
```

Comment:
You don't really need an additional function `rec` here. Just move all this logic and use `data` instead of `obj`. In this way, you would be able to append `ul` to the element straightaway